src/views/Pools/hooks/useUnstakePool.ts

```diff
@@ -1,30 +1,31 @@
 import { useCallback } from 'react'
 import { useWeb3React } from '@web3-react/core'
-import BigNumber from 'bignumber.js'
+import { parseUnits } from 'ethers/lib/utils'
 import { useAppDispatch } from 'state'
 import { updateUserStakedBalance, updateUserBalance, updateUserPendingReward } from 'state/actions'
 import { unstakeFarm } from 'utils/calls'
 import { useMasterchef, useSousChef } from 'hooks/useContract'
-import { BIG_TEN } from 'utils/bigNumber'
 import getGasPrice from 'utils/getGasPrice'
 
-const sousUnstake = async (sousChefContract, amount, decimals) => {
+const sousUnstake = async (sousChefContract: any, amount: string, decimals: number) => {
```

Comment:
We can't use the `Contract` type from `etherjs` here ?

Reply:
Most likely we can but I didn't want to go down the rabbit hole. I also didn't want it not to be types :)